platforms/quarkus/api/README.md

```diff
@@ -13,18 +13,56 @@ mvn quarkus:dev
 
 ## Packaging and running the application
 
+By default, `mvn clean install` produces an executable JAR with the dev Quarkus configuration profile enabled, and in-memory persistence implementation.
+
 The application can be packaged using `mvn package`.
-It produces the `apicurio-studio-platforms-quarkus-api-0.2.47-SNAPSHOT-runner.jar` file in the `/target` directory.
+It produces the `apicurio-studio-api-runner.jar` file in the `/target` directory.
 Be aware that it’s not an _über-jar_ as the dependencies are copied into the `target/lib` directory.
 
-The application is now runnable using `java -jar target/apicurio-studio-platforms-quarkus-api-0.2.47-SNAPSHOT-runner.jar`.
+The application is now runnable using `java -jar target/apicurio-studio-api-runner.jar`.
+
+### Build Options
+
+ - `-Pprod` enables Quarkus's *prod* configuration profile, which uses configuration options suitable for a production environment, 
+   e.g. a higher logging level.
+   
+
+##Runtime Configuration
+
+- In the *dev* mode, the application uses an in-memory H2 database`.
+- In the *prod* mode, you have to provide connection configuration for a PostgreSQL server as follows:
```

Comment:
PostgreSQL or MySQL, right?  Both are supported?

Reply:
If yes, then we should document how to configure the database type.

Reply:
Same change needed for the -ws image as well.

Reply:
Everything should be ok with MySQL as well, just maybe some minor changes needed at config level. I'll check that and document how to configure the database type.